src/app/lib/streamer.js

```diff
@@ -1,501 +1,491 @@
-(function (App) {
+(function (App, WebTorrent) {
```

Comment:
Linter was accusing that WebTorrent wasn't defined on the scope that i used, maybe its wrong.